src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -700,17 +700,28 @@ private void SetLoadedTypesRaw(ILibraryContainer libraryContainer, IEnumerable<N
                 }
 
                 builder.Append("{");
-                builder.AppendFormat(" \"fullyQualifiedName\": \"{0}\", ", entry.FullName);
+                builder.AppendFormat(" \"fullyQualifiedName\": \"{0}\", ", entry.FullyQualifiedName);
                 builder.AppendFormat(" \"iconName\": \"{0}\", ", entry.IconName);
                 builder.AppendFormat(" \"creationName\": \"{0}\", ", entry.CreationName);
                 builder.AppendFormat(" \"itemType\": \"{0}\" ", itemType);
                 builder.Append("}");
             }
 
             builder.Append(@"] }");
+            WriteLibraryToFile(builder);
             libraryContainer.SetLoadedTypesRaw(builder.ToString());
         }
 
+        private void WriteLibraryToFile(System.Text.StringBuilder builder)
+        {
+            // Write the unformatted JSON directly in the docs folder in HostedContents
+            string path = System.IO.Directory.GetCurrentDirectory() + "\\..\\..\\..\\src\\HostedContents\\docs\\";
+            using (StreamWriter sw = new StreamWriter(path + "RawTypeData.json"))
+            {
+                sw.WriteLine(builder.ToString());
+            }
+        }
+
```

Comment:
Can you also include the generated (beautified) `RawTypeData.json` file here? I'll work on a Mac without Dynamo so I won't be able to run this. Thanks! :)